COPYING

```diff
@@ -1,3 +1,13 @@
+Files in this code repository are to be licensed under the LGPLv3, which you'll
+find in the file COPYING-LGPL
+
+However, you'll find some files that carry a license header that assigns them as
+GPLv3, which was the default license for VOLK up to version 3. You can find the
```

Comment:
They should all state GPLv3+. I hope.

Reply:
Well, that should be easy to check?

```
cd volk/kernels/volk
ag -L "GNU General" $(ag -l copyright)
```

yields a list of only two files that have a copyright statement, but aren't GPL, and these two happen to be already LGPL.

There's a bunch of kernel, orc and asm code that doesn't carry a proper copyright note (see attached 
[unlicensed.txt](https://github.com/gnuradio/volk/files/6759814/unlicensed.txt)), but this is the exhaustive list of their committers, found by `( for f in $(ag -L license) ; do; git log  --format="* %aN <%aE>" $f ; done ) |sort -u `: 

* Albin Stigo <albin.stigo@gmail.com>
* Doug Geiger <doug.geiger@bioradiation.net>
* ghostop14 <ghostop14@gmail.com>
* hcab14 <hcab14@gmail.com>
* Johannes Demel <demel@uni-bremen.de>
* Marcus Müller <mueller@kit.edu>
* Nathan West <nathan.west@nrl.navy.mil>
* Nathan West <nathan.west@okstate.edu>

I think we're safe when it comes to kernels.